roseau/load_flow/io/dict.py

```diff
@@ -649,44 +649,60 @@ def v2_to_v3_converter(data: JsonDict) -> JsonDict:
         buses.append(bus_data)
 
     # Rename `uhv` in `up` and `ulv` in `us`
+    # Remove max_power
     old_transformers_params = data.get("transformers_params", [])
     transformers_params = []
     for transformer_param_data in old_transformers_params:
-        if up := transformer_param_data.pop("uhv", None) is not None:
+        if (up := transformer_param_data.pop("uhv", None)) is not None:
             transformer_param_data["up"] = up
-        if us := transformer_param_data.pop("ulv", None) is not None:
+        if (us := transformer_param_data.pop("ulv", None)) is not None:
             transformer_param_data["us"] = us
+        transformer_param_data.pop("max_power", None)
```

Comment:
The value of `max_power` (when not None) should be used to compute the `max_loading` value on transformers that use this Parameters object. Example:
```python
old_tp = {"id": "old_tp1", "sn": 100_000, "max_power": 80_000, ...}
old_tr = {"id": "old_tr1", "params_id":  "old_tp1", ...}

new_tp = {"id": "old_tp1", "sn": 100_000, ...}  # no more max_power
new_tr = {"id": "old_tr1", "params_id":  "old_tp1", "max_loading": old_tp["max_power"]/old_tp["sn"], ...}

```